symplyphysics/laws/dynamics/mechanical_work_during_rotation.py

```diff
@@ -0,0 +1,88 @@
+from sympy import Eq, sin, solve
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    angle_type,
+)
+from symplyphysics.core.expr_comparisons import expr_equals
+from symplyphysics.laws.dynamics import mechanical_work_from_force_and_move as linear_work_law
+from symplyphysics.laws.kinematic import angular_position_is_arc_length_over_radius as angular_position_def
+from symplyphysics.laws.dynamics import torque_due_to_twisting_force as torque_def
+
+# Description
+## When a torque accelerates a rigid body in rotation about a fixed axis, the torque does work on
+## the body. When the torque is constant, the work done on the body is proportional to torque
+## and the angular displacement during the movement of the body.
+
+# Law: W = tau * delta_theta
+## W - work done
+## tau - torque accelerating the body
+## delta_theta - angular displacement of the body
+
+work = Symbol("work", units.energy)
+torque = Symbol("torque", units.force * units.length)
+angular_displacement = Symbol("angular_displacement", angle_type)
+
+law = Eq(work, torque * angular_displacement)
+
+
+# Derive law from non-rotational counterpart.
+## We assume a particle moving along a curved trajectory due to a force F applied to it.
+## The displacement of the particle should be small enough so that the position vector
+## is approximately the same at all points of the trajectory. In that case, the force vector
```

Comment:
Probably worth adding to the conditions, that position vector has constant length. Also force should be constant. And angle between force and position vectors should be constant.
Here https://en.wikipedia.org/wiki/Torque there is a proof using integral of force. It seems it does not require force, angle and radius to be constant.

Reply:
I'll take a look at it